pyproject.toml

```diff
@@ -33,22 +33,21 @@ tzwhere = "^3.0.3"
 backoff = "^1.11.1"
 
 [tool.poetry.dev-dependencies]
-pytest = "^6.2.2"
-requests_mock = "^1.7.0"
-pyyaml = "^6.0"
-black = "^19.10b0"
-pylint = "^2.4.4"
-pydocstyle = "^6.0.0"
-yamllint = "^1.20.0"
-bandit = "^1.6.2"
-invoke = "^1.4.1"
-toml = "0.10.2"
-flake8 = "^4.0.0"
-mypy = "^0.910"
+pytest = "*"
```

Comment:
Can you remind me why we consider it preferable to have wildcard versions for dev dependencies in `pyproject.toml`? It makes it so that inspection of the much-less-human-readable `poetry.lock`is necessary to determine what versions of these dependencies this project is currently developed/tested against, which seems like a loss of useful information to me.

Reply:
We agreed it in dev-standards, and IIRC, the main point was to get, for testing, the latest version of the packages for linting and testing. This is mentioned in issue our internal repo, issue 111

Reply:
My read on that was that it was done for cookiecutters so that *newly created* projects would automatically get the latest available. `poetry.lock` ensures that the version remains pinned thereafter to some exact version, so I don't see any advantage to unpinning dependencies in an existing project.